### ISSUE TITLE
Fails with marked-1.1.0

### ISSUE THREAD

**Issue by sdgathman** [kapouer/marked-man#28]
Unmodified, immediately crashes with:
```
InlineLexer.prototype.html_output = InlineLexer.prototype.output;
            ^

TypeError: Cannot read property 'prototype' of undefined
```

Ok, so that internal name is Lexer now in marked.  Will a simple rename work?
```
var InlineLexer = marked.Lexer;
```
Now the crash is:
```
$ npm test

> marked-man@0.7.0 test /home/stuart/fedora-scm/marked-man/marked-man-0.7.0
> node ./test/compare

(node:2476841) UnhandledPromiseRejectionWarning: TypeError: this.peek is not a function
Please report this to https://github.com/markedjs/marked.
    at Parser.parse (/home/stuart/fedora-scm/marked-man/marked-man-0.7.0/lib/marked-man.js:162:20)
    at Function.parse (/usr/lib/node_modules/marked/src/Parser.js:27:19)
    at Function.marked [as parse] (/usr/lib/node_modules/marked/src/marked.js:108:19)
    at convert (/home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:12:20)
    at check (/home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:87:2)
    at /home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:41:4
    at new Promise (<anonymous>)
    at /home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:40:10
    at Array.map (<anonymous>)
    at /home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:39:20
(node:2476841) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 13)
(node:2476841) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by kapouer** [comment]
Hello, thanks for that report. Will look into it. Ping me in a week or so if i forgot.

**Comment by sdgathman** [comment]
ping

**Comment by kapouer** [comment]
Yep, sorry, fixing this is not that easy and i got busy. Will try again later, but keep pinging !

**Comment by dominikwilkowski** [comment]
I tried using [Selective dependency resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) but that failed as well:
```
"resolutions": {
	"marked-man/marked": "^0.7.0"
},
```

Can you please make `marked` a dependency until this is fixed?

**Comment by bedney** [comment]
@kapouer - Thank you for this package.

Pinging you for a fix of this package so that we can upgrade to `marked-1.1.0`.

Thank you.

**Comment by alinex** [comment]
marked is now at V1.2.0 but then will marked-man use the newer versions?

**Comment by sdgathman** [comment]
Failure with marked-1.2.7:
```
(node:3326462) UnhandledPromiseRejectionWarning: TypeError: this.peek is not a function
Please report this to https://github.com/markedjs/marked.
    at Parser.parse (/home/stuart/fedora-scm/marked-man/marked-man-0.7.0/lib/marked-man.js:162:20)
    at Function.parse (/usr/lib/node_modules/marked/src/Parser.js:27:19)
    at Function.marked [as parse] (/usr/lib/node_modules/marked/src/marked.js:110:19)
    at convert (/home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:12:20)
    at check (/home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:87:2)
    at /home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:41:4
    at new Promise (<anonymous>)
    at /home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:40:10
    at Array.map (<anonymous>)
    at /home/stuart/fedora-scm/marked-man/marked-man-0.7.0/test/compare.js:39:20
(Use `node --trace-warnings ...` to show where the warning was created)
(node:3326462) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 13)
(node:3326462) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by alinex** [comment]
Ping again, will this be fixed any time?

**Comment by bedney** [comment]
@kapouer -

Would making a donation through something like BountySource or Patreon help this along? I can throw a little cash in the pot (or buy you a bottle of your favorite wine or beer) if that would help :-).

Thank you for this project!

**Comment by hildjj** [comment]
ping.

**Comment by kapouer** [comment]
Luckily I am still alive. I've setup a reminder for tomorrow...

**Comment by bedney** [comment]
@kapouer - Any thoughts on fixing this? As per my above message, I'm willing to contribute via Patreon, BountySource, beer etc.

**Comment by anthraxx** [comment]
ping @kapouer

**Comment by kapouer** [comment]
On to it !

**Comment by kapouer** [comment]
Somewhat depending on https://github.com/markedjs/marked/pull/2629 to avoid using another command line options parser.

**Comment by kapouer** [comment]
Fixed in version 1.0.0
